databuilder/extractor/dashboard/redash/redash_dashboard_utils.py

```diff
@@ -1,70 +1,9 @@
 # Copyright Contributors to the Amundsen project.
 # SPDX-License-Identifier: Apache-2.0
 
-from databuilder.rest_api.rest_api_query import RestApiQuery
-
-
-def sort_widgets(widgets):
```

Comment:
any reason with the function moved from top to bottom?

Reply:
Yes: we use the classes defined in this file as inputs and outputs to these utility functions, so need to define those classes first for it to typecheck.